civis/io/_tables.py

```diff
@@ -115,6 +127,23 @@ def read_civis(table, database, columns=None, use_pandas=False,
     if client is None:
         # Instantiate client here in case users provide a (deprecated) api_key
         client = APIClient(api_key=api_key, resources='all')
+
+    if use_pandas == True:
+        if dtype == 'redshift':
+            if NO_NUMPY:
+                raise ImportError("`dtype='redshift'` specified but numpy not installed.")
+            else:
+                table_id = client.get_table_id(table=table, database=database)
+                py_types, date_cols = _redshift_to_py(client, table_id)
+                pd_kwargs = {'parse_dates': date_cols,
+                             'dtype': py_types,
+                             'engine': 'python',
```

Comment:
I defaulted to this because it is more robust and the performance benefits of the C engine seem negligible for our data

Reply:
I'm not comfortable altering this default, especially since it would only apply to a special subset of circumstances and it's hidden from users. Is there a specific problem you're solving with this? Users have the option to modify the "engine" parameter themselves if they so choose.

Reply:
I remember having to change to it but I can't remember why I had to make this change. This is not the first time I have encountered this problem either. If we encounter the error that is raised when the C engine fails it would probably be nice to raise a flag that the user try the python engine